src/SparkFun_External_EEPROM.cpp

```diff
@@ -876,6 +892,9 @@ int ExternalEEPROM::write(uint32_t eepromLocation, const uint8_t *dataToWrite, u
         while (isBusy(settings.deviceAddress) == true) // Poll device's original address, not the modified one
             delayMicroseconds(100); // This shortens the amount of time waiting between writes but hammers the I2C bus
 
+        // Check if we are using Write Protection then disable WP for write access
+        if(settings.usingWP)    digitalWrite(settings.wpPin, LOW);
```

Comment:
We use the built-in Arduino formatter style. Please remove whitespace: usingWP) digitalWrite...